DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan/test_vlan_priority.py

```diff
@@ -0,0 +1,94 @@
+import asyncio
+import pytest
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_setup_streams, tgen_utils_start_traffic,
+    tgen_utils_stop_traffic, tgen_utils_get_traffic_stats,
+    tgen_utils_get_loss, tgen_utils_dev_groups_from_config,
+    tgen_utils_traffic_generator_connect,
+    tgen_utils_get_egress_stats)
+from dent_os_testbed.utils.test_utils.br_utils import (configure_bridge_setup, configure_vlan_setup)
+
+pytestmark = [pytest.mark.suite_functional_vlan,
+              pytest.mark.asyncio,
+              pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")]
+
+
+async def test_vlan_priority_tag(testbed):
+    """
+    Test Name: VLAN priority
+    Test Suite: suite_functional_vlan
+    Test Overview: Test packet forwarding with VLAN priority
+    Test Procedure:
+    1. Create bridge and set links to `Up` state
+    2. Insert links to VLAN's per port_map config
+    3. Map receiving and non receiving dut_ports
+        Port 1 -> tx_port
+        Port 2 -> rx_ports
+    4. Setup packet stream(s) for the broadcast packet with packet vlan and priority specified.
+    Enable egress tracking for the vlan and pcp
```

Comment:
Please distinguish test objectives and implementation. "Enable egress tracking bla bla bla" - this is implementation.  Your test objectives are in steps 6 and 7. Please remove implementation notes.

The same is in step 2. "port_map" this is a reference to the implementation. The objective is "Add two ports to the same VLAN in access mode."